src/components/mixins/CopyAddressMixin.ts

```diff
@@ -3,26 +3,37 @@ import { Component, Mixins } from 'vue-property-decorator';
 import TranslationMixin from './TranslationMixin';
 import { copyToClipboard, delay } from '../../util';
 
-// TODO: make a common solution without 'address' word and use it everywhere
 @Component
 export default class CopyAddressMixin extends Mixins(TranslationMixin) {
+  targetElement: Nullable<EventTarget> = null;
   wasAddressCopied = false;
-  customCopyTooltip: Nullable<string> = null;
-  customCopiedTooltip: Nullable<string> = null;
 
-  async handleCopyAddress(address: string, event?: Event): Promise<void> {
-    event && event.stopImmediatePropagation();
+  async handleCopyAddress(address: string, event?: PointerEvent): Promise<void> {
+    if (event) {
+      event.stopImmediatePropagation();
+      this.targetElement = event.target;
+      if (this.targetElement) {
+        this.targetElement.addEventListener('mouseleave', this.handleMouseleaveListener);
+      }
+    }
     await copyToClipboard(address);
     this.wasAddressCopied = true;
     await delay(1000);
+  }
+
+  async handleMouseleaveListener(): Promise<void> {
+    await delay(1000);
```

Comment:
Isn't this delay is too long? It seems to me that it should be adjusted to the hiding time of the tooltip, probably no more than 500
With 1000 I have time to remove the cursor, hover again, and I see the old message that the address was copied